src/global.css

```diff
@@ -1,2 +1,12 @@
 @import url("https://fonts.googleapis.com/css2?family=IBM+Plex+Serif:ital,wght@0,700;1,700&display=swap");
 @import "~@zanichelli/albe-design-tokens/dist/tokens.css";
```

Comment:
Nope, unfortunately that does not work (just tried).

Reply:
You are right, `::slotted` has limited features. However, I think that the style of a slotted element should be managed by the host application following the directives of the design system, and it would not be good to insert specific styles in a global css file. If we really want to keep this functionality, maybe we can use JS (something like this):
```ts
  connectedCallback() {
    Array.from(this.hostElement.children)
      .find((child) => child.nodeName === "BUTTON" || child.nodeName === "A")
      .querySelector("z-icon").style.marginRight = "var(--space-unit)";
   [...]
  }
```
otherwise, for mere documentation needs we can act on the storybook stories